src/data/emotions.js

```diff
@@ -0,0 +1,141 @@
+const bag = {
```

Comment:
I'd consider changing this file name since you don't have the emotion aspect going on anymore